examples/object_detection/configs/ssd300_vgg_voc_int8.json

```diff
@@ -34,7 +34,8 @@
     "steps": [8, 16, 32, 64, 100, 300],
     "aspect_ratios": [[2], [2, 3], [2, 3], [2, 3], [2], [2]],
     "flip": true,
-    "top_k": 200
+    "top_k": 200,
+    "loss_inference": false
```

Comment:
We actually use this config file for checkpoint validation, so we require the proper metric calculation.

Reply:
@vshampor When the key is enabled, the training loss is evaluated instead of the mean AP. Disabling the key has the exact behavior as previous implementation. Can you elaborate the metric calculation you refer to?

Reply:
I guess @vshampor meant [this](https://github.com/openvinotoolkit/nncf/blob/develop/examples/object_detection/main.py#L189) metric calculation.
It's used in the nightly checkpoint validation defined in test_sota_checkpoints.py. 

But it shouldn't be affected, as you say: the introduced option keeps the previous behavior with calculation actual mAP metric.